Add video after critical CSS inlining (#5775)
GoogleChrome/web.dev: src/site/content/en/blog/aurora-resource-inlining/index.md

```
{% endCompare %}

> After inlining the critical CSS in the HTML we can see that the flickering of the page is gone:

Reviewer: ```suggestion
After inlining the critical CSS in the HTML you will find that the flickering of the page is gone:
```
